[2025-10-23T08:54:49.370+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-23T08:54:49.386+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingest_kg.trigger_load_neo4j_from_gold manual__2025-10-23T08:54:26.579080+00:00 [queued]>
[2025-10-23T08:54:49.389+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingest_kg.trigger_load_neo4j_from_gold manual__2025-10-23T08:54:26.579080+00:00 [queued]>
[2025-10-23T08:54:49.389+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-23T08:54:49.395+0000] {taskinstance.py:2330} INFO - Executing <Task(TriggerDagRunOperator): trigger_load_neo4j_from_gold> on 2025-10-23 08:54:26.579080+00:00
[2025-10-23T08:54:49.398+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ingest_kg', 'trigger_load_neo4j_from_gold', 'manual__2025-10-23T08:54:26.579080+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/ingest_kg.py', '--cfg-path', '/tmp/tmplcu4kqic']
[2025-10-23T08:54:49.400+0000] {standard_task_runner.py:91} INFO - Job 9: Subtask trigger_load_neo4j_from_gold
[2025-10-23T08:54:49.400+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=1131) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-23T08:54:49.400+0000] {standard_task_runner.py:64} INFO - Started process 1132 to run task
[2025-10-23T08:54:49.425+0000] {task_command.py:426} INFO - Running <TaskInstance: ingest_kg.trigger_load_neo4j_from_gold manual__2025-10-23T08:54:26.579080+00:00 [running]> on host c48d41bd1c66
[2025-10-23T08:54:49.518+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ingest_kg' AIRFLOW_CTX_TASK_ID='trigger_load_neo4j_from_gold' AIRFLOW_CTX_EXECUTION_DATE='2025-10-23T08:54:26.579080+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-23T08:54:26.579080+00:00'
[2025-10-23T08:54:49.519+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-23T08:54:51.830+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:175 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.830+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:222 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.831+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1151 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.832+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1248 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.832+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1271 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.833+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1406 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.833+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1439 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.833+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1495 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.833+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1683 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.834+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:1750 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.834+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2025 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.835+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2110 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.835+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2407 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.835+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:2468 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.837+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:3987 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.838+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4096 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.838+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4127 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.838+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4186 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.838+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4225 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.839+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:4275 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.840+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5209 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.840+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5265 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.841+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5302 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.841+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:5358 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.842+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6437 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.842+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6497 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.843+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6538 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.843+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:6588 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.844+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7023 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.844+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7096 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.845+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7267 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.845+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7333 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.845+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7378 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.845+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:7440 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.846+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8222 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.847+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8282 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.847+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8322 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.847+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:8368 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.849+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:9475 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.849+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:9531 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.851+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10532 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.851+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10588 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.851+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10625 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.852+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10680 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.853+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10923 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.853+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:10979 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.853+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11016 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.854+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11072 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.854+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11231 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.854+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11282 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.855+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11420 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.855+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:11477 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.856+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12300 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.856+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12383 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.857+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12439 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.857+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:12524 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.858+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13019 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.858+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13090 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.858+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13139 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.859+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13200 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.860+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13886 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.860+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13941 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.860+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:13977 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.861+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14026 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.861+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14318 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.861+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14388 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.862+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14436 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.862+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14492 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.862+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14744 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.863+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:14810 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.864+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15415 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.864+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15482 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.864+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15527 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.864+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15578 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.865+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15897 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.865+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:15953 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.866+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16617 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.867+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16674 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.867+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16712 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.867+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16780 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.868+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16932 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.868+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:16988 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.868+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17025 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.869+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17090 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.869+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17228 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.869+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:17284 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.873+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20234 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.873+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20289 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.874+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20862 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.874+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20921 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.874+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:20960 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.875+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21014 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.878+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21316 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.878+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21377 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.879+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21418 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.879+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21478 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.879+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21612 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.880+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:21668 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.881+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22155 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.881+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22218 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.882+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22559 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.882+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22626 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.882+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22671 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.882+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:22722 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.883+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23061 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.883+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23128 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.884+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23173 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.884+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23224 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.884+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23511 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.885+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23567 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.885+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:23968 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.886+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:24018 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.887+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25343 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.888+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25398 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.888+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25434 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.888+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:25485 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.890+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:26723 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.890+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:26779 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.891+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27121 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.891+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27177 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.892+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27461 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.892+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:27517 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.893+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28053 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.893+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28109 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.894+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28536 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.894+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28592 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.895+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28932 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.895+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:28982 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.897+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:30727 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.897+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:30783 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.898+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31296 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.898+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31344 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.899+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31617 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.899+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:31673 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.900+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32120 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.900+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32176 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.901+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32339 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.901+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32395 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.901+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32432 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.902+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:32487 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.903+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33502 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.903+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33558 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.904+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33869 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.904+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33928 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.904+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:33967 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.905+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34021 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.906+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34739 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.906+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34806 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.906+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34852 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.907+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:34908 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.909+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:36915 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.909+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:36971 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.910+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37417 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.910+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37473 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.911+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37845 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.911+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:37901 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.912+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38208 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.912+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38264 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.912+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38301 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.913+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38351 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.913+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38690 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.914+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:38758 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.914+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39141 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.914+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39192 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.915+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39419 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.915+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39487 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.916+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39683 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.916+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:39738 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.916+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:40019 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.917+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:40084 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.918+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41138 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.918+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41196 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.919+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41359 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.919+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41415 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.919+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41452 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.920+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41507 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.920+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41875 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.921+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:41947 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.925+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42441 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.926+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42497 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.926+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42534 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.926+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42590 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.927+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42627 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.927+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:42692 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.928+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43020 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.928+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43079 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.929+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43118 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.929+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43168 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.930+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43514 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.930+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43573 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.931+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43612 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.931+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43666 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.932+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:43989 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.932+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44045 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.933+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44386 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.934+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:44442 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.951+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45003 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.951+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45058 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.953+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45444 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.953+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:45499 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.955+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46042 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.956+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46098 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.958+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46702 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.958+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:46758 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.959+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47000 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.959+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47050 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.960+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47648 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.961+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:47704 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.962+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48113 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.963+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48190 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.963+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48244 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.963+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:48321 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.967+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49499 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.967+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49555 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.968+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:49932 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.968+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50002 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.969+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50052 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.969+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50107 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.970+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50379 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.970+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50449 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.970+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50499 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.971+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50554 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.971+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50590 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.971+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50660 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.972+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50710 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.972+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:50770 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.973+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51046 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.973+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51116 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.974+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51166 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.974+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51226 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.975+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51485 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.976+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51540 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.976+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51787 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.977+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51851 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.977+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51895 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.977+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:51958 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.978+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52212 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.979+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52275 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.979+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52463 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.980+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:52532 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.981+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53040 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.981+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53111 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.982+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53334 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.982+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53407 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.984+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:53961 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.984+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54059 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.985+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54322 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.985+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:54398 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.987+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55022 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.987+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55078 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.988+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55310 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.988+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:55365 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.990+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56196 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.990+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56248 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.991+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56508 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.991+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:56564 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.992+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57087 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.993+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57147 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.993+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57187 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.993+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57237 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.995+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57560 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.995+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57620 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.995+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57660 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:51.996+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:57710 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.001+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59106 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.002+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59162 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.002+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59316 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.002+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59396 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.003+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59452 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.004+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59535 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.004+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59594 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.004+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59677 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.004+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59733 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.005+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:59818 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.007+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60639 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.008+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60722 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.008+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60778 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.008+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:60863 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.009+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61167 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.010+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61250 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.010+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61306 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.010+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61387 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.011+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61696 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.012+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61752 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.012+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61954 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.013+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:61992 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.014+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:62660 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.015+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:62715 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.018+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:63882 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.018+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:63946 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.020+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64582 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.020+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64674 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.021+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:64940 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.021+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65014 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.022+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65117 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.022+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65199 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.023+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65385 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.023+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65441 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.024+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65874 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.025+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:65950 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.025+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66003 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.025+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66087 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.027+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66686 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.028+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66779 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.028+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66918 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.028+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:66964 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.030+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:67735 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.030+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:67791 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.031+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68043 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.032+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68097 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.033+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68532 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.034+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/azure/synapse/artifacts/models/_models_py3.py:68588 SyntaxWarning: invalid escape sequence '\d'
[2025-10-23T08:54:52.381+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-23T08:54:52.388+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ingest_kg, task_id=trigger_load_neo4j_from_gold, run_id=manual__2025-10-23T08:54:26.579080+00:00, execution_date=20251023T085426, start_date=20251023T085449, end_date=20251023T085452
[2025-10-23T08:54:52.439+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-23T08:54:52.464+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-23T08:54:52.465+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
